contract/__manifest__.py

```diff
@@ -9,7 +9,7 @@
 
 {
     'name': 'Recurring - Contracts Management',
-    'version': '12.0.4.1.0',
+    'version': '12.0.4.1.1',
     'category': 'Contract Management',
     'license': 'AGPL-3',
     'author': "OpenERP SA, "
```

Comment:
Are you sure of this? Is it a legacy porting ?

Reply:
Original code (although not too much of it now) was from `account_analytic_analysis` module.